commands/reserves/reservableitem.js

```diff
@@ -0,0 +1,70 @@
+const { Op } = require("sequelize");
+
+// Add a new reservable item to the raid db
+exports.run = async function(client, message, args) {
+
+    if (!client.permission.manageReserves(message.member)) {
+		return message.channel.send('**Error:** You need permission to be able to add items to the reservable items list.');
+	}
+
+
+    if (!args[0]) {
+        return message.channel.send('**Error:** Correct usage is ``+reservableitem add 30024 TK Mantle of the Elven Kings``');
+    };
+
+
+    let addOrRemove = args.shift().toLowerCase();
+
+    if (addOrRemove !== 'add' && addOrRemove !== 'remove') {
+        return message.channel.send('**Error:** Must specify **ADD** or **REMOVE**!')
+    };
+
+
+    let record = {
+        'itemID': parseInt(args.shift()),
+        'raid': args.shift().toUpperCase(),
+        'name': args.join(' ')
+    };
+
+
+    let validRaidList = ['MC', 'BWL', 'Ony', 'ZG', 'AQ20', 'AQ40', 'Naxx', 'GL', 'Kara', 'ML', 'SSC', 'TK'];

```

Comment:
Can we future proof this by moving this to a function, and adding in the next raid tier?  (BT + MH)